sos/report/reporting.py

```diff
@@ -168,7 +168,7 @@ def unicode(self):
         i = 0
         plugcount = len(self.report_data)
         for section_name, _ in self.report_data:
-            line += f"  {section_name}"
+            line += self.PLUGLISTITEM % {'name': section_name}
```

Comment:
This removes leading spaces - isnt it a problem?

Reply:
I didn't see an issue when I checked the result, but I'll add the spaces to keep the same format

Reply:
The space is added when the variable PLUGLISTITEM is defined, via `PLUGLISTITEM = "  %(name)s"`